nava/functions.py

```diff
@@ -82,28 +82,35 @@ def quoter(sound_path, *args, **kwargs):
     return quoter
 
 
-def __play_win(sound_path, async_mode=False):
+def __play_win(sound_path, async_mode=False, loop=False):
     """
     Play sound in Windows.
 
     :param sound_path: sound path
     :type sound_path: str
     :param async_mode: async mode flag
     :type async_mode: bool
+    :param loop: sound loop flag
+    :type loop: bool
     :return: None or sound id
     """
     import winsound
-    play_flags = winsound.SND_FILENAME | (async_mode & winsound.SND_ASYNC)
+    play_flags = \
+        winsound.SND_FILENAME | \
+        (async_mode & winsound.SND_ASYNC)
+    if loop:
+        play_flags = play_flags | winsound.SND_LOOP
```

Comment:
@sepandhaghighi What was the problem with following?
```python
    play_flags = \
        winsound.SND_FILENAME | \
        (async_mode & winsound.SND_ASYNC) | \
        (loop & winsound.SND_LOOP)
```

Reply:
I'm not sure what the issue is, but it didn't work in this mode.